en/components/general/wpf_to_angular_guide/angular_pipes.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Angular Pipes
+_description: Learn how to transform data in Angular similar to converters in WPF.
+_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular, Pipe, Converter
+---
+
+## Transforming Data With Angular Pipes
+
+In WPF in order to transform data from one value to another you are going to use [IValueConverter](https://docs.microsoft.com/en-us/dotnet/api/system.windows.data.ivalueconverter?view=netframework-4.8). To transform data in an Angular application, we are going to use a [Pipe](https://angular.io/api/core/Pipe). The pipe is very similar to the WPF converter. It takes data as an input and then transforms that data into a desired output for display.
```

Comment:
missing comma - In WPF in order to transform data from one value to another, you are going

Reply:
Addressed.